Header header
Lidar_Point[] points

float64 max_distance
float64 xmin
float64 xmax
float64 ymin
float64 ymax
float64 zmin
float64 zmax
